corehq/apps/accounting/async_handlers.py

```diff
@@ -159,10 +159,11 @@ class Select2BillingInfoHandler(BaseSelect2AsyncHandler):
 
     @property
     def country_response(self):
-        from django_countries.countries import COUNTRIES
+        from django_countries.data import COUNTRIES
+        countries = COUNTRIES.items()
```

Comment:
I think we'll want to also sort `countries` by country name ascending, since order isn't guaranteed by the dictionary and it looks like this is being referenced from a ui where displaying in order would be important.